composer.json

```diff
@@ -14,7 +14,7 @@
         "php": ">=5.4",
         "php-http/httplug": "^1.1",
         "php-http/message-factory": "^1.0",
-        "php-http/message": "^1.2",
+        "php-http/message": "^1.6@dev",
```

Comment:
You should remove "@dev"

Possible add "minimun-stabillity:dev" and "prefer-dist "